blog/tooltip-w-teorii-i-praktyce/index.md

```diff
@@ -0,0 +1,168 @@
+  szkolenia użytkowników.
+
+## Gdzie warto je stosować?
+
+Wykorzystanie tooltipów może być dość wszechstronne w różnych biznesach, ale
+skupmy się na kilku typowych w branży technologicznej. Można do nich zaliczyć
+między innymi:
+
+- wycieczki wdrożeniowe – gdy użytkownik po raz pierwszy loguje się do
+  aplikacji, może odbyć wycieczkę po aplikacji lub jej funkcjach. Wyświetlane w
+  aplikacji podpowiedzi pokazują każdy aspekt danej funkcji i dostarczają
+  informacji o tym, co robi dana ikona. Wariantem wycieczek mogą być
+  interaktywne przewodniki, czyli podpowiedzi pojawiają się dopiero po
+  interakcji użytkownika z poprzednią podpowiedzią, dzięki czemu są bardziej
+  angażujące niż ogólna wycieczka po aplikacji.
+- wprowadzanie nowych funkcji – z których użytkownicy jeszcze nie korzystali -
+  za pomocą podpowiedzi możesz wyróżnić funkcje, które nie były jeszcze używane
+  i nauczyć użytkowników, jak z nich korzystać
+- zapowiadanie nowych funkcji i zmian w produktach – podpowiedzi doskonale
+  nadają się do ogłaszania zmian w aplikacji czy dodawania nowych funkcji.
+- zapewnienie pomocy, gdy użytkownik utknie w martwym punkcie – jeśli użytkownik
+  nie może ruszyć dalej, tooltipy mogą pomoc w wykonaniu konkretnych czynności
+  lub skierować użytkownika do źródeł „samopomocy”. Jeśli użytkownicy potrzebują
+  więcej informacji, wówczas mogą skorzystać z podpowiedzi zamiast kontaktować
+  się z pomocą techniczną.
+
+## Co jest ważne podczas tworzenia tooltipów?
+
+Chociaż tooltipy nie są nowością w sieci, często są one niewłaściwie stosowane
+lub zaimplementowane. Na co zatem warto zwrócić uwagę przy ich tworzeniu?
+
+- dodawajcie krótki tekst – tooltipy z oczywistym lub zbędnym tekstem nie
+  przynoszą korzyści użytkownikom. Jeśli nie możecie wymyślić pomocnej treści,
+  nie oferujcie podpowiedzi. Dodana podpowiedź powinna być samowystarczalna,
+  czyli powinna być zwięźle opisana i nie blokować powiązanych treści… w
+  przeciwnym razie zmarnujecie czas użytkownika, który będzie miał pecha
+  aktywować tę podpowiedź ☹.
+- używajcie podpowiedzi ze strzałkami – gdy elementy znajdują się blisko siebie
+  strzałki pomagają uniknąć nieporozumień.
+- używajcie tooltipów konsekwentnie w całej aplikacji. Niektóre elementy
+  aplikacji są trudne do znalezienia, ponieważ brakuje im wizualnych wskazówek.
+  Jeśli podpowiedzi są wyświetlane nieregularnie w całej aplikacji, użytkownicy
+  mogą nigdy ich nie odkryć. Ważne jest, aby być konsekwentnym i zapewniać
+  podpowiedzi dla wszystkich elementów projektu, a nie tylko wybranych. Jeśli
+  tylko niektóre elementy wymagają dodatkowych wyjaśnień, użyj wyskakujących
+  podpowiedzi dla tych elementów.
+- używajcie podpowiedzi dla niejednoznacznych ikon – ikony mają pewien poziom
+  niejednoznaczności, warto więc przemyśleć czy użyć tooltipy dla wszystkich
+  ikon lub zapewnić użytkownikowi wersje opisową przycisku.
+- zadbajcie o to, aby tooltipy miały umiarkowane tło i czcionkę – umiarkowany
```

Comment:
co to znaczy "umiarkowane tlo i czcionka"? pozniej jest informacja o kontrascie. 

Moze zamiast "umiarkowane" napisz "odpowiednie" czy "czytelne"?